omnigibson/object_states/kinematics_mixin.py

```diff
@@ -29,7 +29,7 @@ def cache_info(self, get_value_args):
         info[self.obj] = {"q": self.obj.states[Joint].get_value(), "p": self.obj.states[Pose].get_value()}
         for arg in get_value_args:
             if isinstance(arg, StatefulObject):
-                info[arg] = {"q": self.obj.states[Joint].get_value(), "p": arg.states[Pose].get_value()}
+                info[arg] = {"q": arg.states[Joint].get_value(), "p": arg.states[Pose].get_value()}
```

Comment:
@yyf20001230 could this be related to your joint state error?

Reply:
Yes.